app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    cmd, source_file, target_file = command.split()
+    if cmd != "cp" or source_file == target_file:
```

Comment:
You're checking if the command is 'cp', which is good, but you also need to check that the source file exists before trying to open it. You can use the `os.path.exists` function for this. Also, it's a good practice to give a feedback to the user about the error, so consider adding a print statement or raising an error with a descriptive message when the command is not 'cp' or the source file does not exist.